SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -640,6 +657,12 @@ void USpatialReceiver::CleanupDeletedEntity(Worker_EntityId EntityId)
 	NetDriver->RemoveActorChannel(EntityId);
 }
 
+void USpatialReceiver::ClearRPCsOnEntityCreation(const Worker_EntityId& EntityId)
+{
+	Worker_ComponentUpdate Update = RPCsOnEntityCreation::CreateClearFieldsUpdate();
+	NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
```

Comment:
Hmm, this function sends an update but exists in the SpatialReceiver. Could it exist in the SpatialSender instead?

Reply:
Moved to Sender